### PR TITLE
fix: app integration takes to wrong org

### PR DESCRIPTION
**Pull request by Alamin-Ahammed**
image

See the above image, clicking this connect button takes to the wrong org. It was for not updateing the wp system info and some necessary data correctly. Here it is fixed. 

Dependecy: with frontend (app-frontend) td
https://github.com/thrivedesk/app-frontend/pull/1171

### CODE REVIEW

File: resources/js/admin.js [hunk 1]
```diff
@@ -309,6 +311,36 @@ jQuery(document).ready(($) => {
...
+						if (result.isConfirmed) {
+							jQuery
+							.post(thrivedesk.ajax_url, {
+								action: 'thrivedesk_system_info',
+								data: {
+									td_helpdesk_api_key: apiKey,
+								},
+							})
+							.success(function (response) {
+								console.log('data system info updted')
```
Comment (Rijoanul-Shanto) [inline resources/js/admin.js:324]: remove console log

Comment (Alamin-Ahammed) [reply resources/js/admin.js:324]: but it is intensional. It will help us to debug.

**Comment by shamsbd71** [comment]
[wp-thrivedesk_v2.0.8.zip](https://github.com/user-attachments/files/17617487/wp-thrivedesk_v2.0.8.zip)

**Review by Rijoanul-Shanto — CHANGES_REQUESTED** [review]
add changelogs

**Comment by Alamin-Ahammed** [comment]
> add changelogs

already added but need to modify: 
image

**Pull request merged**